flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGenerator.java

```diff
@@ -928,14 +1011,8 @@ private StreamConfig createJobVertex(Integer streamNodeId, OperatorChainInfo cha
         return new StreamConfig(jobVertex.getConfiguration());
```

Comment:
Can we move the above logic of reset parallelism to setVertexParallelismForDynamicGraphIfNecessary?

Reply:
Done